docs/components/concepts/access-control/authorizations.md

```diff
@@ -0,0 +1,150 @@
+- When enabled: no access by default - explicit authorization required
+- No concept of deny rules - absence of permission equals no access
+- Access is enforced in both web applications and API requests
+
+### Key Components
+
+1. **Authorizations**
+
+   - Assign permissions to identities for specific resources
+   - Example: User 'jonny' is authorized to create new users
+   - Example: Group 'marketing' is authorized to delete the Group 'sales'
+
+2. **Owners**
+
+   - Types: users, groups, roles, and mapping rules
+   - Authorizations can be assigned to any owner type
+
+3. **Permissions**
+
+   - Define allowed interactions with resources
+   - Specific to resource types
+
+4. **Resources**
+   - Entities users interact with
+   - Each has specific available permissions
+
+## Configuration
+
+### SaaS Configuration
+
+Authorizations can be activated/deactivated on a per-cluster level by:
+
+- Organization Admins
+- Organization Owners
+
+### Self-Managed Configuration
+
+Enable authorizations in your deployment using:
+
+```yaml
+camunda.security.authorizations.enabled: true
+```
+
+## Resources and Permissions
+
+### Available Resources
+
+The following table lists all resources that support authorization in Camunda 8 Orchestration Cluster, along with the available permissions per resource. This applies to Camunda 8 Orchestration Cluster (Zeebe, Operate, Tasklist, Orchestration Cluster APIs).
+
+| Resource Type                        | Resource Key Example                                    | Supported Permissions                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| **Application**                      | `Operate`, `Tasklist`, `Identity`, `*`                  | `ACCESS`                                                                                                                   |
+| **Authorization**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Batch**                            | `*`                                                     | `CREATE`, `READ`, `DELETE`                                                                                                 |
+| **Decision Definition**              | `*`, `decisionId: order_decision`                       | `CREATE_DECISION_INSTANCE`, `READ_DECISION_DEFINITION`, `READ_DECISION_INSTANCE`, `DELETE_DECISION_INSTANCE`               |
+| **Decision Requirements Definition** | `*`, `decisionRequirementsDefinitionId: order_decision` | `CREATE`, `READ`, `UPDATE`                                                                                                 |
```

Comment:
🔧 We don't have `CREATE` here. In the code we've got:

```java
  DECISION_REQUIREMENTS_DEFINITION(
      PermissionType.READ, PermissionType.UPDATE, PermissionType.DELETE),
```

It's a bit odd that we have all these. Creation works through the `RESOURCE` resource type. Same with deletion. Updating of a DRD isn't something we support at all as an update is a deployment of a new verison. Imo only `READ` is relevant here.